pytorch_lightning/overrides/torch_distributed.py

```diff
@@ -1,99 +0,0 @@
-import logging
```

Comment:
shouldn't this have been done in another PR?

Reply:
I've done it all together in a single unit of work because the PR is not too long and it helps validate the CI runs properly.

But if you insist, I could split it